test/hotspot/jtreg/vmTestbase/gc/ArrayJuggle/Juggle01/Juggle01.java

```diff
@@ -67,7 +72,9 @@ public void run() {
                 log.debug("Garbage producer: " + garbageProducer);
                 log.debug("Memory strategy: " + memoryStrategy);
                 long memory = runParams.getTestMemory();
-                int objectCount = memoryStrategy.getCount(memory);
+                // Keep object count to 4, intention is to juggle
+                // not to overwhelm gc
+                int objectCount = 4;
```

Comment:
I don't see how it is used.

Reply:
array = new Object[objectCount - 1];
This is the array where juggling happens. Limiting the object count limits the number of objects created.